docs/triaging.md

```diff
@@ -0,0 +1,46 @@
+Issue Triaging
+==============
+
+OmnICU uses GitHub for tracking feature requests and work items.
+
+- All issues should have a type label.
+    - [Query: issues needing a type](https://github.com/unicode-org/omnicu/issues?q=is%3Aissue+-label%3Abug+-label%3Acore+-label%3Adocumentation+-label%3Aenhancement+-label%3Aquestion+-label%3Ainvalid)
+- All *open* or *backlog* issues should have and assignee or **help wanted** label.
+    - [Query: open or backlog issues needing assignee or help wanted](https://github.com/unicode-org/omnicu/issues?q=is%3Aissue+is%3Aopen+-label%3A%22help+wanted%22+no%3Aassignee+)
+- All *closed* issues should have a resolution, linked pull request, or the **question** type.
+    - [Query: closed issues needing resolution or linked PR](https://github.com/unicode-org/omnicu/issues?q=is%3Aissue+is%3Aclosed+-linked%3Apr+-label%3Aquestion+-label%3Abacklog)
+
+## Fields
+
+### Type
+
+The following labels are types; every issue should have one:
+
+- **bug** = a defect in existing code.
+- **core** = a high-priority improvement or new feature.
```

Comment:
Can it be then:
core-feature or feature?

The only difference vs enhancement is priority/impact.

Reply:
Done.  Labels seem to use spaces so I renamed "core" to "core feature"